sunpy/map/tests/test_plotting.py

```diff
@@ -47,6 +47,22 @@ def test_plot_aia171(aia171_test_map):
     aia171_test_map.plot()
 
 
+@figure_test
+def test_plot_rotated_aia171(aia171_test_map):
+    # Check that plotting a rotated map and a rectangle works as expected
+
+    # Set rotation metadata
+    aia171_test_map.meta['CROTA2'] = 45
```

Comment:
Maybe this line should call `.rotate()` instead of modifying the header?

Reply:
I think this is fine, because it's going to be a lot quicker than actually rotating the array, and it tests the code just fine.